control/control.openSUSE.xml

```diff
@@ -186,6 +186,8 @@ textdomain="control"
                 </window_manager>
             </window_managers>
         </upgrade>
+        <!-- boo#1124590 - Ensure correct product is selected -->
+        <select_product>openSUSE-MicroOS</select_product>
```

Comment:
You want the openSUSE control.xml to select MicroOS?

That seems..wrong to me.

Reply:
Has been fixed now.